AppCenter/AppCenter/Internals/Channel/MSOneCollectorChannelDelegate.m

```diff
@@ -81,6 +75,36 @@ - (void)channel:(id<MSChannelProtocol>)__unused channel prepareLog:(id<MSLog>)lo
   }
 }
 
+- (void)channel:(id<MSChannelProtocol>)channel
+     didPrepareLog:(id<MSLog>)log
```

Comment:
Not really about this PR, but this callback (and all others in `MSChannelProtocol`, really) should probably use `id<MSChannelUnitProtocol>` as the type for parameter `channel`. (It's what the `shouldFilter` callback does).

Reply:
Actually there are too many delegates and callbacks to have un understandable delegate pattern. According to me, at this stage we should go with NSNotification. But that's another story.

Reply:
+1